common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -1353,9 +1367,6 @@
         "unicorn/no-unreadable-iife": [
             "error"
         ],
-        "unicorn/no-unsafe-regex": [
```

Comment:
Does this need to be disabled?

Reply:
It's now deprecated: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/6d15a02d48de7ecfc38d0683a8487b2f937d83a0/docs/deprecated-rules.md?plain=1#L19

Reply:
Probably worth adding a comment to the PR description.